benchmarks/src/main/java/io/druid/benchmark/datagen/BenchmarkColumnSchema.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.benchmark.datagen;
+
+import io.druid.segment.column.ValueType;
+
+import java.util.List;
+
+public class BenchmarkColumnSchema
+{
+  public enum ValueDistribution
+  {
+    SEQUENTIAL,
+    UNIFORM,
+    NORMAL,
+    ZIPF,
+    ENUMERATED
+  }
+
+  private final String name;
+  private final Integer cardinality;
+  private final ValueType type;
+  private final boolean isMetric;
+  private final ValueDistribution distributionType;
+  private final int rowSize;
+  private final List<Object> enumeratedValues;
+  private final List<Double> enumeratedProbabilities;
+  private final Double nullProbability;
+
+  public BenchmarkColumnSchema(
+      String name,
+      ValueType type,
+      boolean isMetric,
+      ValueDistribution distributionType,
+      int rowSize,
+      int cardinality,
```

Comment:
@xvrl 

I'm only supporting discrete distributions right now, the "normal" distribution isn't a true normal distribution, it's also discrete (it rounds the real sample values from an underlying normal distribution)

I went with this approach so that the 'cardinality' property imposes an upper limit on the cardinality of possible column values.